l10nru.properties

```diff
@@ -1,15 +1,45 @@
 category=Localization
 description=Language pack for Russian
 homepageUrl=https://github.com/1c-syntax/sonar-l10n-ru
-publicVersions=7.9.1,8.0.1,8.1,8.2,8.3.1,8.4.1,8.5
-archivedVersions=1.0,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,1.10,8.0,8.3,8.4
+publicVersions=7.9.1,8.0.2,8.1.1,8.2.1,8.3.1,8.4.1,8.5.1,8.6
+archivedVersions=1.0,1.1,1.2,1.3,1.4,1.5,1.6,1.7,1.8,1.9,1.10,8.0,8.3,8.4,8.5,8.0.1,8.1,8.2
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-l10n-ru-plugin
 
+8.6.description=Support SonarQube 8.6
+8.6.date=2021-01-01
+8.6.sqVersions=[8.6,LATEST]
+8.6.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v8.6/sonar-l10n-ru-plugin-8.6.jar
+8.6.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v8.6
+
+8.2.1.description=Small translation fixes
+8.2.1.date=2021-01-01
+8.2.1.sqVersions=[8.2,8.2]
```

Comment:
FYI, no need to use range syntax when there's no range. This could be just `8.2`